src/components/FormComponents/ApaPreview.jsx

```diff
@@ -49,6 +49,7 @@ export function generateCitation(record,language, format) {
       issued: { "date-parts": [[datePublished || created]] },
       publisher: publishers.join(", "),
       DOI: datasetIdentifier.replace(/https?:\/\/doi\.org\//, ""),
+      version: record.edition,
```

Comment:
any issue with wording consistency here? all references to the field are for `edition` but the citation indicates `version`.

Reply:
its fine for now, we will review what the citation looks like and can change if needed. It was suggested that we prefix version with a  'v' if the library doesn't do that already